src/services/processTx.js

```diff
@@ -108,7 +108,11 @@ async function newPendingTran(tx, protocol) {
             // smart contract call hence the asset must be the token name
             to = data.params[0].value;
             pillarId = await client.getAsync(to);
-            [, { value }] = data.params;
+            value = data.params[1].value;
```

Comment:
Is necessary this change?

Reply:
I didnt realize that this change was related to linter, will amend it.